app/views/groups/create.js.erb

```diff
@@ -2,4 +2,6 @@ var added_group = $("<%= j(render 'groups/group_card', group: @group)%>");
 
 $("#new_group_modal").fadeOut();
 
+$("#group_form")[0].reset();
```

Comment:
Oh, I guess do need the `[0]` if you're resetting a form. TIL—

You can [alternatively use](https://stackoverflow.com/a/24505433)

```suggestion
$("#group_form").trigger("reset");
```